packages/runtime-core/src/apiLifecycle.ts

```diff
@@ -49,56 +49,56 @@ function injectHook(
 }
 
 export function onBeforeMount(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.BEFORE_MOUNT, hook, target)
 }
 
 export function onMounted(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.MOUNTED, hook, target)
 }
 
 export function onBeforeUpdate(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.BEFORE_UPDATE, hook, target)
 }
 
 export function onUpdated(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.UPDATED, hook, target)
 }
 
 export function onBeforeUnmount(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.BEFORE_UNMOUNT, hook, target)
 }
 
 export function onUnmounted(
-  hook: Function,
+  hook: () => void,
   target: ComponentInternalInstance | null = currentInstance
 ) {
   injectHook(LifecycleHooks.UNMOUNTED, hook, target)
 }
 
 export function onRenderTriggered(
-  hook: Function,
+  hook: () => void,
```

Comment:
signature is wrong here. `onRenderTriggered` and `onRenderTracked` receives a debugger event argument.